src/ui/public/directives/__tests__/config.js

```diff
@@ -0,0 +1,59 @@
+var ngMock = require('ngMock');
+var $ = require('jquery');
+var assign = require('lodash').assign;
+var expect = require('expect.js');
+
+describe('Config Directive', function () {
+
+  var build = function () {};
+
+  beforeEach(ngMock.module('kibana', function ($compileProvider) {
+    var renderCount = 0;
+    $compileProvider.directive('renderCounter', function () {
+      return {
+        link: function ($scope, $el) {
+          $el.html(++renderCount);
+        }
+      };
+    });
+  }));
+
+  beforeEach(ngMock.inject(function ($compile, $rootScope) {
+
+    build = function (attrs, scopeVars) {
+      var $el = $('<config>').attr(attrs);
+      var $scope = $rootScope.$new();
+      assign($scope, scopeVars || {});
+      $compile($el)($scope);
+      $scope.$digest();
+      return $el;
+    };
+
+  }));
+
+  it('renders it\'s config template', function () {
```

Comment:
I'm an extreme fella :stuck_out_tongue:  I usually break the contraction to avoid double quotes